app/goal_routes.py

```diff
@@ -0,0 +1,101 @@
+from app import db
+from flask import Blueprint, make_response, jsonify, abort, request 
+from app.models.goal import Goal
+from app.models.task import Task
+from app.routes import validate_model
+
+
+goals_bp = Blueprint("goal", __name__, url_prefix = "/goals")
+
+@goals_bp.route("", methods = ["POST"])
+def post_one_goal():
+    request_body = request.get_json()
+    try:
+        request_body["title"]
+    except:
+        abort(make_response({"details": "Invalid data"}, 400))
+    goal = Goal(
+        title = request_body["title"]
+    )
+
+    db.session.add(goal)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model